airflow/providers/apache/spark/hooks/spark_sql.py

```diff
@@ -44,15 +44,19 @@ class SparkSqlHook(BaseHook):
     :type executor_memory: str
     :param keytab: Full path to the file that contains the keytab
     :type keytab: str
-    :param master: spark://host:port, mesos://host:port, yarn, or local
+    :param master: (Deprecated) spark://host:port, mesos://host:port, yarn, or local
+        This parameter has been deprecated. Master and connection parameters (such as YARN queue)
+        are determined by the conn_id parameter.
     :type master: str
     :param name: Name of the job.
     :type name: str
     :param num_executors: Number of executors to launch
     :type num_executors: int
     :param verbose: Whether to pass the verbose flag to spark-sql
     :type verbose: bool
-    :param yarn_queue: The YARN queue to submit to (Default: "default")
+    :param yarn_queue: (Deprecated) The YARN queue to submit to (Default: "default")
```

Comment:
You haven't just deprecated this -- you've made it do nothing at all.

Please make this argument still be respected if passed